src/components/ExclusivePostListing/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from "react";
+import { ScrollView } from "react-native";
+
+import ContentExclusivePost from './components/ContentExclusivePost';
+
+//import styles from "./styles";
```

Comment:
Esse comentário acho que tem que descomentar, não?

Reply:
Então, eu não utilizei estilos nesse componente porque ele somente lista outros componentes já estilizados. Mas decidi deixar esse CSS em prontidão aí porque talvez quando for adaptado para carregamento paginado ele poderá ser útil.